src/Data/RdsData/Decode/Array.hs

```diff
@@ -195,6 +198,13 @@ days = do
     Just d -> pure d
     Nothing -> DecodeArray \_ -> Left "Failed to decode Day"
 
+ulids :: DecodeArray [ULID]
+ulids = do
+  ts <- texts
+  case traverse CONV.textToUlid ts of
+    Right u -> pure u
+    Left msg -> DecodeArray \_ -> Left $ "Failed to decode UUID: " <> msg
+
 uuids :: DecodeArray [UUID]
```

Comment:
Not necessary but I might put a comment by the encoder and decoder that steers consumers towards `ULID`s.

Reply:
Done